app/helpers/state_file/pattern_matching_helper.rb

```diff
@@ -3,8 +3,8 @@ module PatternMatchingHelper
     def contains_po_box(address)
       return false if address.blank?
 
-      # Regex tested at https://regex101.com/r/SksZmO/1
-      pattern = /(?i)p(?:[.\s]*)o(?:[.\s]*)(?:b(?:[.\s]*)(?:o(?:[.\s]*)x)?)?/
+      # Regex tested at https://regex101.com/r/V64RgG/1
+      pattern = /(?i)\bp(?:[.\s]*)o(?:[.\s]*)(?:b(?:[.\s]*)(?:o(?:[.\s]*)x)?)?\b/
```

Comment:
want to document here that the previous regex was too sensitive:
i.e. `333 Potomac Street` or `ponderbox` or `P obo xes` were considered to contain po box